### ISSUE TITLE
smart contracts for art (NFT)

### ISSUE THREAD

**Issue by monperrus** [KTH/programmable-society#4]

**Comment by monperrus** [comment]
CryptoKitties KittyCore https://etherscan.io/token/0x06012c8cf97bead5deae237070f9587f8e7a266d#code

**Comment by monperrus** [comment]
Cryptopunks https://etherscan.io/token/0xb47e3cd837ddf8e4c57f05d70ab865de6e193bbb#code

**Comment by monperrus** [comment]
Bored Ape Yacht Club https://etherscan.io/token/0xbc4ca0eda7647a8ab7c2061c2e118a18a936f13d#code

**Comment by monperrus** [comment]
Unit-on-Chain is a podcast series from Unit London. Here, we offer a ground for critical discussions for artists from the Web3 ecosystem. https://anchor.fm/unit-london

**Comment by monperrus** [comment]
Mutagen contract: https://etherscan.io/token/0xDf9E0684F15e60cFCC646ACffB02d97D2D5a1a67#code

**Comment by monperrus** [comment]
> Founded in 2021, Fingerprints is a global DAO dedicated to collecting, curating, and creating significant blockchain art

https://fingerprintsdao.xyz/

**Comment by monperrus** [comment]
> Art Blocks is a platform for generative art
https://www.artblocks.io/

**Comment by bbaudry** [comment]
fxhash is a platform for generative NFTs
https://www.fxhash.xyz/

**Comment by bbaudry** [comment]
curated digital art and NFTs
https://www.niio.com/site/

**Comment by bbaudry** [comment]
Feral File commissions curated exhibitions of digital artwork 
They have a native cryptocurrency, USDC—you can also use Wyre to transfer ETH, XTZ, and BTC into your Feral File wallet.
https://feralfile.com/

**Comment by bbaudry** [comment]
KnownOrigin is a NFT marketplace, built on Ethereum
https://knownorigin.io/

**Comment by bbaudry** [comment]
What Can DAOs Do For You?

Terence Trouillot speaks to Ruth Catlow, Rhea Myers, Penny Rafferty and Bhavik Singh on how DAOs can support artists, create a more equitable art world and promote social play within the digital sphere 

https://www.frieze.com/article/what-can-daos-do-for-you

**Comment by bbaudry** [comment]
Teia is a community owned evolution of the Tezos NFT Marketplace Hic et Nunc.
https://github.com/teia-community

**Comment by monperrus** [comment]
shackled: on-chain 3D rendering engine
https://shackled.spectra.art/#/blog
http://arxiv.org/pdf/2301.06702v1

**Comment by monperrus** [comment]
Kohi: on-chain generative art rendering
https://kohi.art/onchain
https://github.com/kohiart/kohi-composer/
https://etherscan.io/address/0xdf6e32d85d17e907e0da157fab7c12788e7161da#code

**Comment by monperrus** [comment]
fiveoutofnine is the first 100% on-chain chess engine, where minters play against the smart contract. Each move is minted as an NFT, and accompanied by a generative art piece.
https://www.fiveoutofnine.com/

**Comment by bbaudry** [comment]
the contracts that fuel [gen.art](https://gen.art/)
https://gen.art/contracts

**Comment by bbaudry** [comment]
no code NFT
https://bueno.art/

**Comment by bbaudry** [comment]
DAO for music events
https://mirror.xyz/superficialmachines.eth

**Comment by monperrus** [comment]
Tiles: Generative art from any Ethereum address
https://tiles.wtf/

**Comment by bbaudry** [comment]
the first generation of WebAssembly Music entirely stored on the NEAR-blockchain
https://petersalomonsen.com/

**Comment by monperrus** [comment]
Definition and Detection of Defects in NFT Smart Contracts
https://arxiv.org/pdf/2305.15829.pdf

**Comment by Jacarte** [comment]
[Encrypto.art](https://encrypto.art/) 
> The new tool for securing your artwork on the Symbol Blockchain

**Comment by bbaudry** [comment]
[What do blockchain technologies imply for digital creative industries?](https://onlinelibrary.wiley.com/doi/abs/10.1111/caim.12456)

**Comment by bbaudry** [comment]
Open-source NFT contracts used by Buildship 
https://github.com/artgenexyz/nft-contracts

**Comment by monperrus** [comment]
What Is Generative Art? A Quintessentially Modern Art Form
https://www.sothebys.com/en/articles/generative-art-a-quintessentially-modern-art-form
https://etherscan.io/address/0xe034bb2b1b9471e11cf1a0a9199a156fb227aa5d#code

**Comment by bbaudry** [comment]
Definition and Detection of Defects in NFT Smart Contracts
http://arxiv.org/abs/2305.15829

**Comment by bbaudry** [comment]
The open-source coding and art community for the web3 space
https://hashlips.io/

**Comment by monperrus** [comment]
> Now an acknowledged pioneer whose work has graced the auction room at Sotheby’s, Myers embarked on her first art projects focusing on blockchain tech in 2011, making her one of the first artists to enter into creative, speculative and conceptual engagement with ‘the new internet’.

https://rhea.art/art/index.html

**Comment by monperrus** [comment]
> Primavera De Filippi, Kevin McCoy, and Rhea Myers discuss the importance of smart contracts to the new creator economy 

https://www.rightclicksave.com/article/nfts-and-the-law-of-the-blockchain

**Comment by monperrus** [comment]
2023 lumen prize crypto art award
https://www.lumenprize.com/2023-crypto-award-longlist
https://www.lumenprize.com/2023-crypto-art-award-shortlist

**Comment by monperrus** [comment]
> The NFT Canon is a go-to resource for artists and creators, developers, corporations and institutions, communities, and other organizations seeking to understand or do more with non-fungible tokens.

https://a16zcrypto.com/posts/article/nft-canon/

**Comment by monperrus** [comment]
the seaport protocol and contracts for NFT marketplaces
* doc: https://docs.opensea.io/reference/seaport-overview
* contracts: https://github.com/ProjectOpenSea/seaport

**Comment by monperrus** [comment]
Moma postcard
https://www.moma.org/calendar/exhibitions/5618

Built on top of the Autonomy Digital Art Wallet https://github.com/bitmark-inc/autonomy-client

**Comment by bbaudry** [comment]
a fully compliant implementation of IERC721 with significant gas savings for minting multiple NFTs in a single transaction
https://github.com/chiru-labs/ERC721A

**Comment by bbaudry** [comment]
Non-Fungible Token (NFT): Overview, Evaluation, Opportunities and Challenges. 2021
https://arxiv.org/abs/2105.07447

**Comment by bbaudry** [comment]
Your very own Smart Contract for NFTs
https://medium.com/@mondoir/your-very-own-smart-contract-for-nfts-b068b7d65e5d

**Comment by bbaudry** [comment]
Non Fungible Tokens in Digital Art: a case study with Ethereum. MSc  thesis 2021
https://www.politesi.polimi.it/retrieve/8794674c-fd5a-4ac3-bcf1-a7cfc2c8f5df/Davide_Clementi_Master_Thesis.pdf

**Comment by bbaudry** [comment]
[Creating with blockchain technology: the" provably rare" possibilities of crypto art](https://open.library.ubc.ca/soa/cIRcle/collections/ubctheses/24/items/1.0370991) MSc thesis 2018

**Comment by bbaudry** [comment]
A Brief History of the Creeps & Weirdos
https://nfts.wtf/a-brief-history-of-the-creeps-weirdos/

**Comment by monperrus** [comment]
Manifold a platform for easy creation of NFTS 
* https://manifold.xyz/
* contracts https://github.com/manifoldxyz

**Comment by bbaudry** [comment]
The Autoglyphs are a highly optimized generative algorithm capable of creating billions of unique artworks, wrapped inside an ERC-721 interface. While ERC-721 is the standard for “non-fungible tokens” (something that the [CryptoPunks](https://www.larvalabs.com/cryptopunks) helped define), it is generally used to manage ownership of digital items stored elsewhere. The key difference with the Autoglyphs is that the art is inside the contract itself, it is literally “art on the blockchain.”
https://www.larvalabs.com/autoglyphs

**Comment by bbaudry** [comment]
Money Making Opportunity (MMO) is a smart contract-based coordination game in which 1474 participants must collaborate to distribute 45.736981 ETH
https://moneymakingopportunity.eth.limo/
code: https://github.com/scyclow/moneymakingopportunity

**Comment by bbaudry** [comment]
Do NFTs’ Owners Really Possess their Assets? A First Look at the NFT-to-Asset Connection Fragility
https://dl.acm.org/doi/10.1145/3543507.3583281

**Comment by bbaudry** [comment]
Characterizing the Solana NFT Ecosystem
https://dl.acm.org/doi/10.1145/3589335.3651478

**Comment by bbaudry** [comment]
Smart contracts for the [ready-made token](https://www.distributedgallery.com/artworks/rmt/)
https://github.com/distributedgallery/readymadetoken

**Comment by bbaudry** [comment]
[Contextualizing the concepts and values of Crypto Art and NFTs](https://mlo.art/research/publications/)

**Comment by bbaudry** [comment]
True On-Chain Artwork: A Four-Year Journey
https://www.artblocks.io/curated/articles/true-on-chain-preservation-a-four-year-journey

**Comment by monperrus** [comment]
Notes from the Avant NFT Underground
https://www.rightclicksave.com/article/notes-from-the-avant-nft-underground-avant-gay-solana

**Comment by bbaudry** [comment]
The NFT Apocalypse
https://www.rightclicksave.com/article/the-nft-apocalypse
